packages/datadog-plugin-tedious/test/index.spec.js

```diff
@@ -13,7 +13,7 @@ describe('Plugin', () => {
   let connection
   let connectionIsClosed
 
-  withVersions('tedious', 'tedious', version => {
+  withVersions('tedious', 'tedious', {'<16': '<=15'}, version => {
```

Comment:
Currently this relies on the `Object.keys()` order which means we should put more restrictive (higher versions) first. This means the the following situation could happen:

```js
{'<16': '<=15', '<14': '<=13'} // works as expected 
{'<14': '<=13', '<16': '<=15'} // Node.js v16 would run against v13 of the library not v15
```

Any suggestions on how to 'sort' the list based on semver restrictiveness?

I don't think it's a big issue but will be in the future if we get hugely complex version matrixes.